### ISSUE TITLE
Create the V-REP simulation model

### ISSUE THREAD

**Issue by rsanz** [Ingenia-SE/T-Groundair#2]
Create the **V-REP model** to support the development of the system. Note that this model will not be the only model in the project. The folder "models" shall contain all models (SysML, V-REP, Simulink, etc).

**Comment by rsanz** [comment]
Create the Gazebo model that will be the **final delivered** system due to curfew. Note that this model will not be the only model in the project. The folder "models" shall contain all models (SysML, Gazebo, Simulink, etc).

**Comment by rsanz** [comment]
What is the status of the complete system simulation model?

**Comment by jruro** [comment]
Por el memento solo tenemos el entorno de ENV. La parte de UGV tenemos el
modelo, pero el control aún no, y la parte de UAV tenemos el modelo y el
control pero nos está dando problemas el control porque solo hay un paquete
mediante el cual se puede controlar y no lo conseguimos. Nos reuniremos
entre hoy y mañana para ver si conseguimos avanzar.

Un saludo,

Juan.

**Comment by rsanz** [comment]
¿Es necesario tener el control para tener un modelo de simulación completo?, ¿Geometría y física sin control?

Salud !!!

**Comment by jruro** [comment]
No, los modelos ya los tenemos, tanto de todo el entorno, como de los
robots, el UAV y el UGV. El problema es que esos modelos son modelos, no
hemos implementado el control en ellos, se supone que los diversos grupos
lo están estudiando. He hablado con el responsable de UAV y esta tarde
coordinaré el desarrollo a ver si lo conseguimos.

Por otro lado, si quieres ya te puedo crear el escenario completo, con todo
incluido. Así que dime si necesitas algún tipo de entregable, o alguna
reunión en la que te pueda poner al día.

Saludos y gracias,

Juan.

**Comment by jruro** [comment]
Hola Ricardo. Después de 4h que hemos estado Enrique y yo con la
integración, hemos conseguido integrar el Drone con el entorno y con el
UGV. Te envío este correo para decirte que nosotros podemos ser los
coordinadores y que hemos hecho la mayor parte del trabajo de momento. A
ver si a partir de ahora van las cosas mas rodadas y el resto de grupos nos
van diciendo algo.

Dime qué te parece y si hay algo más que podamos hacer.

Un saludo y gracias.

Juan.

**Comment by rsanz** [comment]
Hola Juan, @kikemagan,

Good news :-))

En principio Juan, tu eras el coordinador de la simulación (junto con Rafa Menéndez @rafamen8) y Enrique @kikemagan lo era del control (junto con Jorge Hunter @jgehunter ). Así que para el acoplamiento control-simulación es bueno que Enrique y tú os coordinéis.

¿Dónde está el modelo integrado para que el que quiera pueda usarlo? 
¿Podeis preparar un pequeño README en markdown sobre cómo usarlo?

**Comment by jruro** [comment]
Buenas Ricardo, lo tendrás todo disponible a lo largo de la semana que viene, seguramente para el martes/miércoles ya estarán todos los paquetes de simulación y control.

Saludos y feliz fin de semana.

Juan.
